.circleci/config.yml

```diff
@@ -97,6 +97,28 @@ jobs:
           paths:
             - ~/spree/vendor/bundle
 
+  brakeman:
+    <<: *defaults_3_0
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - spree-brakeman-{{ .Branch }}
+            - spree-brakeman
+      - run:
+          name: Ensure Brakeman Installed
+          command: |
+            gem install brakeman -v 4.3.1 && bundle install
+            ./bin/build-ci.rb install
+      - run:
+          name: Brakeman Spree API
```

Comment:
@aleksandarpetrushev we need this to be also run against spree core

Reply:
@damianlegawiec Trying to set it up for api first, then I will do core